multinet/uploaders/csv.py

```diff
@@ -46,9 +51,15 @@ def validate_csv(rows):
                 detail.append({"fields": fields, "row": i + 2})
 
         if detail:
-            return {"error": "syntax", "detail": detail}
+            data_errors.append({"error": "syntax", "detail": detail})
+    else:
+        # Unsupported Table, error since we don't know what's coming in
+        data_errors.append({"error": "unsupported"})
```

Comment:
For clarity, my change from the other PR was this `{"error": "format", "detail": "Invalid header format"}`

Reply:
I was trying to be consistent with the other responses and they all seemed to be one word. I think if we change this, we should update them all. I think that parsing these errors in the clients will be easier with a single word response

Reply:
I'd agree. Another thing we could do to standardize better is add custom 400 codes. This is more something for #143 though, I think what you have is fine.